### PR TITLE
Add Socialhead by OneCommerce

### PR DESCRIPTION
**Pull request by seia-soto**
Found on https://github.com/ghostery/trackerdb/issues/132.

- https://www.crunchbase.com/organization/onecommerce-025e
- https://socialhead.io/
- https://onecommerce.io/
- https://onecommerce.io/about-us/

### CODE REVIEW

**Comment by philipp-classen** [comment]
The linked ticket (https://github.com/ghostery/trackerdb/issues/132) is about sdks.automizely-analytics.com, which should be that one here: https://www.similarweb.com/website/automizely.com/

Regarding Socialhead, do you know websites where it is being used?

**Comment by seia-soto** [comment]
I found the tracking request heading to widget-api.socialhead.io on the same website. I assume they're for advertisement and momentum.

**Comment by philipp-classen** [comment]
I'm not familiar with Socialhead, but reading through their offerings, OnePixel seems to be their tracking pixel (Site Analytics), but socialwidget looks more like "Social Media" category.

onepixel

socialwidget

**Comment by seia-soto** [comment]
I think I still need to find what websites are using their features. However, I think it's possible to add multiple definitions like: socialhead_onepixel and socialhead_socialwidget.

If I recall correctly, onepixel was hosted on the same domain. If we regard the main functionality caused other tracking requests to be loaded is social widget, I think we can change the category.

**Comment by seia-soto** [comment]
Pushed a commit!

File: db/patterns/socialhead.eno [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: socialhead
```
Comment (philipp-classen) [inline db/patterns/socialhead.eno:1]: Do you think we should rename it to "Socialwidget" or "Socialhead's Socialwidget"? To distinguish it from other Socialhead offerings such as OnePixel?

Comment (seia-soto) [reply db/patterns/socialhead.eno:1]: What about "Socialhead" + "Socialwidget" just like "Google" + "Analytics"?

Comment (philipp-classen) [reply db/patterns/socialhead.eno:1]: Yes, we can use `Socialhead Socialwidget`

Comment (seia-soto) [reply db/patterns/socialhead.eno]: Thanks! Done.

**Comment by seia-soto** [comment]
Done!

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**